app/src/main/java/com/codesoom/assignment/domain/ProductCommand.java

```diff
@@ -0,0 +1,35 @@
+package com.codesoom.assignment.domain;
+
+import lombok.Builder;
+import lombok.Getter;
+import lombok.ToString;
+@lombok.Generated
```

Comment:
```suggestion
import lombok.ToString;

@lombok.Generated
```

NIT : 별거아니지만 빈칸이 있는 것이 좋겠어요!